srv/modules/runners/rebuild.py

```diff
@@ -64,9 +66,12 @@ def __init__(self, *args, **kwargs):
         __master_opts__['quiet'] = True
         self.qrunner = salt.runner.RunnerClient(__master_opts__)
         self.minions = self._minions(*args)
-        self.operation = kwargs.get('operation', 'osd.remove')
-        self.aggressive = kwargs.get('aggressive', False)
+        self.kwargs = kwargs
         self.skipped = []
+        if 'preserve_ids' in kwargs and kwargs['preserve_ids']:
```

Comment:
for future reference:

```suggestion
        if kwargs.get('preserve_ids', False): # or whatever the default should be 
```